index.js

```diff
@@ -25,7 +25,7 @@ async function compareLastModifiedTime(stream, sourceFile, targetPath) {
 	// TODO: Use the `stat` `bigint` option when targeting Node.js 10 and Gulp supports it
 	const targetStat = await stat(targetPath);
 
-	if (sourceFile.stat && Math.floor(sourceFile.stat.mtimeMs) > Math.floor(targetStat.mtimeMs)) {
```

Comment:
Can you add a code comment about why it's using `floor` instead of `ceil` and link to the Node.js issue?

Reply:
Yes absolutely 👍

Reply:
Let me know if that is too wordy or whatever